src/test/java/com/amazon/sqs/javamessaging/AmazonSQSExtendedClientTest.java

```diff
@@ -172,7 +206,7 @@ public void testWhenMessageBatchIsSentThenOnlyMessagesLargerThanThresholdAreStor
                 700_000,
                 800_000,
                 900_000,
-                200_000,
+                20_000,
```

Comment:
Needed to make this smaller so that the messages not sent to s3 did not themselves exceed the batch size in this instance.